lib/primitives/tx.js

```diff
@@ -1893,7 +1893,6 @@ class TX extends bio.Struct {
    * size as it is framing (exposed on return value as `witness`).
    * @private
    * @param {BufferWriter} bw
-   * @returns {Sizes}
```

Comment:
Well this function still returns something doesn't it?
Probably `@returns {BufferWriter}` or `@returns {Buffer}`

Reply:
Yup, it returns a `{BufferWriter}`, I was gonna add a return statement but then it looked like the BufferReader didn't have a `@returns` either so I left it like that, though on second thoughts, I should fix that.